frontend/app/routes/login.tsx

```diff
@@ -87,7 +98,8 @@ export default function Login() {
               display: "flex",
               justifyContent: "center",
               width: "60vw",
-              marginTop: "50px",
+              marginTop: "30px",
+              marginBottom: "20px",
```

Comment:
try using the mui way of margins, so just numbers instead of `px`

It will get turned into $8x$ pixels by default, where 8 is the number you give.

We can also create a file with all those magic numbers so we can reuse them.